scanners/access-logs.sh

```diff
@@ -14,7 +14,7 @@ declare -a success_regexes;
 #
 # [GET or POST] [space] [any url prefix] /vpns/ [any url fragment].xml [space] HTTP/1.1" [space] [200 or 304]
 #                                         ^^^^ maybe encoded
-success_regexes[0]="(GET|POST)\s[^\s]*/(v|%76)(p|%70)(n|%6[Ee])(s|%73)/[^\s]*\.xml\sHTTP/1\.1\"\s(200|304)";
+success_regexes[0]="\]\s\"[A-Z]{3,7}\s[^\s]*/(v|%76)(p|%70)(n|%6e)(s|%73)/[^\s]*\.(x|%[57]8)(m|%[46]d)(l|%[46]c)[^\s]*\sHTTP/1\.1\"\s(200|304)";
```

Comment:
I like the addition of the encoded `.xml` extension. Ditto with relaxing the method.

Let's get some test data added as well so that we can verify this works as claimed. Then, happy to merge!

Reply:
test
will delete this right after